README.md

```diff
@@ -1 +1,77 @@
 # ArDrive Price Calculator
+
+## Developer Setup
+
+Follow these steps to get the developer environment up and running:
+
+### Install Yarn 2
+
+The ArDrive Price Calculator uses Yarn 2, so install the latest version with the [yarn installation instructions][yarn-install]. In most cases:
+
+```shell
+# Brew:
+brew install yarn
+
+# Or with NPM:
+npm install -g yarn
+```
+
+We also use husky to manage the git commit hooks that help to improve the quality of our commits. Without installing husky, you risk committing non-compliant code to the repository.
+
+Using husky triggers two pre-commit hooks. The first will run `lint-staged` on each staged file, which includes running prettier to format, eslint for linting, and also tsc-files for quickly checking that TypeScript can compile the code. The second hook will test the codebase prior to committing, ensuring that all tests must pass.
+
+To enable hooks locally, you will need to run:
+
+```shell
+yarn husky install
+```
+
+### NVM
+
+This repository uses NVM and an `.nvmrc` file to lock the Node version to the current version used by `ardrive-core-js`.
+
+Note for Windows: **We recommend using WSL** for setting up NVM on Windows using the [instructions described here][wsl-install]
+
+Follow these steps to get NVM up and running on your system:
+
+1. Install NVM using [these installation instructions][nvm-install].
+2. Navigate to this project's root directory
+3. Ensure that the correct version of Node is installed by performing: `nvm install`
+4. **Every time you start a new terminal session**, you should set the correct version of Node by running: `nvm use`
```

Comment:
Nit: "... or switch to this project from another NPM project, ...". Not critical though.